pymc/distributions/timeseries.py

```diff
@@ -107,10 +109,10 @@ def rng_fn(
             init_size = (*size, 1)
             steps_size = (*size, steps)
 
-        init_val = rng.normal(init, sigma, size=init_size)
+        init = np.reshape(init, init_size)
```

Comment:
Why is no random draw needed here? Is it because we defer to the random method of the init dist?

Reply:
Yes, we assume init already has the right number of random draws (that's why we resize it in `dist`)